app/presenters/queries/content_item_presenter.rb

```diff
@@ -143,6 +145,11 @@ def parse_results(results)
             int_columns.each { |c| parse_int_column(result, c) }
             parse_state_history(result)
 
+            if self.include_warnings
```

Comment:
Yeah, the wrapper method is really just a convenience to indicate that it is a boolean, and is a convention, it just makes me pause when reading it. 
Although saying that, the style of having attr_accessors and assigning these instances to self is not very common so not really a big deal here.